android/src/play/java/com/dooboolab/RNIap/RNIapModule.java

```diff
@@ -199,21 +199,13 @@ private void consumeItems(
 
   @ReactMethod
   public void refreshItems(final Promise promise) {
-    //    Purchase.PurchasesResult purchasesResult =
-    // billingClient.queryPurchases(BillingClient.SkuType.INAPP);
-    //    purchasesResult.getPurchasesList();
     ensureConnection(
         promise,
         new Runnable() {
           @Override
           public void run() {
-            final WritableNativeArray array = new WritableNativeArray();
             Purchase.PurchasesResult result =
                 billingClient.queryPurchases(BillingClient.SkuType.INAPP);
-            if (result == null) {
-              promise.reject("refreshItem", "No results for query");
-              return;
-            }
```

Comment:
Won't we have any downside removing this? If then, can we also remove 
```
Purchase.PurchasesResult result = billingClient.queryPurchases(BillingClient.SkuType.INAPP);
```

Reply:
Hello @hyochan, no it won't have any effect at all since `array` is never used and `result` is non-nullable so that check is also not needed. We do however use result so we cannot remove it's assignment